widgetssdk/src/main/res/layout/chat_view.xml

```diff
@@ -15,7 +15,7 @@
         android:id="@+id/chat_recycler_view"
         android:layout_width="match_parent"
         android:layout_height="0dp"
-        app:layout_constraintBottom_toTopOf="@id/divider_view"
+        app:layout_constraintBottom_toTopOf="@id/operator_typing_animation_view"
```

Comment:
when view is gone does it behave ok?

Reply:
Depends on what is okay. I did not see anything weird.